docs/podman-run.1.md

```diff
@@ -289,20 +289,27 @@ Add additional groups to run as
 
 **--healthcheck**=""
 
-Set or alter a healthcheck for a container.  The value must be of the format of:
+Set or alter a healthcheck command for a container. The command is a command to be executed inside your
+container that determines your container health.  The command is required for other healthcheck options
+to be applied.  A value of `none` disables existing healthchecks.
 
- `[OPTIONS] CMD command`
+**--healthcheck-interval**=""
 
- where options can be any of the follow:
- * --interval=DURATION (default: 30s)
- * --timeout=DURATION (default: 30s)
- * --start-period=DURATION (default: 0s)
- * --retries=N (default: 3)
+Set an interval for the healthchecks (a value of `disable` results in no automatic timer setup) (default "30s")
 
-Note: options are *not* required.
+**--healthcheck-retries=**
 
-The command is a command to be executed inside your container that determines your container health.  The
-command is required.
+The number of retries allowed before a healthcheck is considered to be unhealthy.  The default value is `3`.
+
+**--healthcheck-start-period**=""
+
+The initialization time needed for a container to bootstrap. The value can be expressed in time format like
+`2m3s`.  The default value is `0s`
+
+**--healthcheck-timeout**=""
+
+The maximum time allowed to complete the healthcheck before an interval is considered failed.  Like start-period, the
+value can be expressed in a time format such as `1m22s`.  The default value is `30s`.
```

Comment:
Not yours, but since you're in there, can you alphabetize the `--http-proxy` option in the list?  It somehow managed to get into the middle of the '--e*' options.